pages/api/landingPage.js

```diff
@@ -0,0 +1,59 @@
+import prisma from "client";
+
+export default async function handler(req, res) {
+  if (req.method === "POST") {
+    return await addContent(req, res);
+  } else if (req.method == "GET") {
+    return await readContent(req, res);
+  } else if (req.method == "PATCH") {
+    return await updateContent(req, res);
+  } else {
+    return res
+      .status(405)
+      .json({ message: "Method not allowed", success: false });
+  }
+}
+
+async function updateContent(req, res) {
+  try {
+    const { id, ...data } = req.body;
+    const updatedUser = await prisma.landing_Page.update({
+      where: { id },
+      data,
+    });
+    res.status(200).json(updatedUser);
+  } catch (error) {
+    console.log(error);
+    res.status(400).json({ error: "Failed to update user data." });
```

Comment:
I'm assuming this is just some copy/paste errors but please correct the following

1. Change references to `user` to `content`
2. Instead of updating "data" we should just be updating the `content` value in the database. The Id, user, and creationDate should be unchanged

Reply:
what do you mean, how?

Reply:
this functionality was not implemented/tested/used in this PR.